CMakeLists.txt

```diff
@@ -60,6 +59,36 @@ endif()
 
 add_compile_options(-Wall -Wextra -Wpedantic)
 
+option(IRODS_ENABLE_ADDRESS_SANITIZER "Enables detection of memory leaks and other features provided by Address Sanitizer." OFF)
+if (IRODS_ENABLE_ADDRESS_SANITIZER)
+  # The following options are compiled into all binaries and allow them to start even
+  # when an ODR violation is detected.
+  #
+  # Make sure the correct llvm-symbolizer binary is available to Address Sanitizer. This binary
+  # allows debug symbols to be reported appropriately. There are two ways to do this:
+  #
+  #     export PATH=/opt/irods-externals/clang13.0.0-0/bin:$PATH
+  #
+  # - or -
+  #
+  #     export ASAN_SYMBOLIZER_PATH=/opt/irods-externals/clang13.0.0-0/bin/llvm-symbolizer
+  #
+  #add_compile_definitions(IRODS_ADDRESS_SANITIZER_DEFAULT_OPTIONS="log_path=/tmp/irods_client_http_api_asan_output:detect_odr_violation=1")
```

Comment:
how does this commented line align with the initial comment above?

Reply:
That is one of the things I'm planning on revisiting before merging this. So, the short answer is, I haven't spent the time thinking about it yet. I also haven't confirmed this works. Will do that soon though.

Reply:
Gotcha.   Resolve whenever you feel ready.

Reply:
Done